hawkular-wildfly-monitor/src/main/assembly/module-assembly.xml

```diff
@@ -39,6 +39,14 @@
       <directoryMode>0755</directoryMode>
     </fileSet>
   </fileSets>
+  <files>
+    <file>
+      <source>${project.basedir}/src/main/assembly/subsystem.xml</source>
+      <outputDirectory>/${moduleDir}</outputDirectory>
+      <destName>subsystem-snippet.xml</destName>
+    </file>
+  </files>
+
 
   <dependencySets>
     <dependencySet>
```

Comment:
This is a bit tricky: either we supply that file in the installer, but then we need to update the installer when it changes. Or the installer needs to unpack the module.zip!org/hawkular/agent/monitor/main/hawkular-monitor.jar to find it. Or we require downloading from a running (web) server which then needs to serve that. 
For automation purposes, I think the possibility for offline install is valid.